public/apps/configuration/panels/test/get-started.test.tsx

```diff
@@ -64,9 +69,9 @@ describe('Get started (landing page)', () => {
     const component = shallow(
       <GetStarted
         coreStart={mockCoreStart as any}
-        navigation={{} as any}
```

Comment:
Yes I would also have liked this, however, when I was writing the initial code to introduce this feature, I found the naming impossible to accept. Basically, before this change, the SecurityPluginStartDependencies was called "navigation". Thus, to get the datasource plugin, we would need to call something like `navigation.dataSource` and to get the navigation dependency it would need to be `navigation.navigation`. I decided to rename it for ease of reading, even if it bloated up the size of this PR.